prow/plugins/trigger/generic-comment_test.go

```diff
@@ -62,10 +62,12 @@ type testcase struct {
 	Presubmits           map[string][]config.Presubmit
 	IssueLabels          []string
 	IgnoreOkToTest       bool
-	ElideSkippedContexts bool
+	ElideSkippedContexts *bool
 }
 
 func TestHandleGenericComment(t *testing.T) {
+	truth := true
+	lies := false
```

Comment:
`var lies bool`

Reply:
I feel like in this case this is clearer, but 
/shrug